MAUI/Popup/layout-customizations.md

```diff
@@ -0,0 +1,451 @@
+                    VerticalOptions="Start" HorizontalOptions="Center" 
+                    Clicked="ClickToShowPopup_Clicked" />
+            <sfPopup:SfPopup x:Name="sfPopup" 
+                             AppearanceMode="OneButton" 
+                             ShowFooter="True">
+            </sfPopup:SfPopup>
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="10" %}
+using Syncfusion.Maui.Popup;
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        sfPopup.ShowFooter = true;
+        // Setting the AppearanceMode as OneButton
+        sfPopup.AppearanceMode = PopupButtonAppearanceMode.OneButton;
+    }
+
+    private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+    {
+        sfPopup.Show();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with accept button](Images/layout-customizations/maui-popup-appearance-mode-one-button.png)
+
+In the following code example, set the `SfPopup.AppearanceMode` property as `TwoButton` which displays both Accept button and Decline button in the footer view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="11" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupMauiLayout.MainPage">
+     <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+                    VerticalOptions="Start" HorizontalOptions="Center" 
+                    Clicked="ClickToShowPopup_Clicked" />
+            <sfPopup:SfPopup x:Name="sfPopup" 
+                             AppearanceMode="TwoButton" 
+                             ShowFooter="True">
+            </sfPopup:SfPopup>
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="10" %}
+using Syncfusion.Maui.Popup;
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        sfPopup.ShowFooter = true;
+        // Setting the AppearanceMode as TwoButton
+        sfPopup.AppearanceMode = PopupButtonAppearanceMode.TwoButton;
+    }
+
+    private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+    {
+        sfPopup.Show();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with accept and decline buttons](Images/layout-customizations/maui-popup-appearance-mode-two-button.png)
+
+## Popup region customization
+
+### Disable header
+
+You can display the Popup without header by using the property `SfPopup.ShowHeader`. Default value is true. Find the code example of the same below.

```

Comment:
Display the Popup without a header by using the property `SfPopup.ShowHeader`. The default value is true. Find the code example of the same as follows.